utils/ecl-diff/ecl-diff-full.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, unicorn/no-reduce, no-param-reassign */
```

Comment:
Repeated ignore rules. Consider making changes to configurations to remove this copy/paste.